--- mozldap-6.0.3/mozilla/directory/c-sdk/ldap/libraries/libldap/Makefile.in.orig	2007-02-24 18:05:00.000000000 +0100
+++ mozldap-6.0.3/mozilla/directory/c-sdk/ldap/libraries/libldap/Makefile.in	2007-03-17 20:45:23.142469884 +0100
@@ -203,7 +203,7 @@
 endif
 
 ifeq ($(OS_ARCH), Linux)
-EXTRA_LIBS = -L$(dist_libdir) -l$(LBER_LIBNAME)
+EXTRA_LIBS = -L$(dist_libdir) -l$(LBER_LIBNAME) -lpthread
 endif
 
 ifeq ($(OS_ARCH), WINNT)
