--- mozldap-6.0.6/mozilla/directory/c-sdk/configure.in.orig	2008-09-11 16:38:34.000000000 +0200
+++ mozldap-6.0.6/mozilla/directory/c-sdk/configure.in	2008-10-01 00:33:21.974636497 +0200
@@ -2621,7 +2621,13 @@
 dnl ========================================================
 dnl Generate output files.
 dnl ========================================================
-MAKEFILES="
+
+dnl since configure won't handle 2 levels of directory
+dnl structure for us..
+mkdir ldap > /dev/null 2>&1
+mkdir ldap/clients > /dev/null 2>&1
+
+AC_OUTPUT([
 Makefile
 config/Makefile
 config/autoconf.mk
@@ -2636,13 +2642,4 @@
 ldap/libraries/libiutil/Makefile
 ldap/libraries/libssldap/Makefile
 ldap/libraries/libutil/Makefile
-"
-
-dnl since configure won't handle 2 levels of directory
-dnl structure for us..
-mkdir ldap > /dev/null 2>&1
-mkdir ldap/clients > /dev/null 2>&1
-
-echo $MAKEFILES > unallmakefiles
-
-AC_OUTPUT([$MAKEFILES])
+])
