--- mozldap-6.0.5/mozilla/directory/c-sdk/configure.in.orig	2007-09-17 19:46:23.000000000 +0200
+++ mozldap-6.0.5/mozilla/directory/c-sdk/configure.in	2008-06-19 00:42:22.810015812 +0200
@@ -2625,7 +2625,13 @@
 dnl ========================================================
 dnl Generate output files.
 dnl ========================================================
-MAKEFILES="
+
+dnl since configure won't handle 2 levels of directory
+dnl structure for us..
+mkdir ldap > /dev/null 2>&1
+mkdir ldap/clients > /dev/null 2>&1
+
+AC_OUTPUT([
 Makefile
 config/Makefile
 config/autoconf.mk
@@ -2640,11 +2646,4 @@
 ldap/libraries/libiutil/Makefile
 ldap/libraries/libssldap/Makefile
 ldap/libraries/libutil/Makefile
-"
-
-dnl since configure won't handle 2 levels of directory
-dnl structure for us..
-mkdir ldap > /dev/null 2>&1
-mkdir ldap/clients > /dev/null 2>&1
-
-AC_OUTPUT([$MAKEFILES])
+])
